src/Payments/Payment.php

```diff
@@ -0,0 +1,487 @@
+use Webit\WFirmaSDK\PaymentCashboxes\PaymentCashboxId;
+use Webit\WFirmaSDK\Series\SeriesId;
+use Webit\WFirmaSDK\Tags\TagIds;
+use JMS\Serializer\Annotation as JMS;
+
+/**
+ * @JMS\XmlRoot("payment")
+ */
+final class Payment extends DateAwareEntity {
+    /**
+     * @var string
+     * @JMS\Type("string")
+     * @JMS\SerializedName("object_name")
+     * @JMS\Groups({"request", "response"})
+     */
+    private $objectName;
+
+    /**
+     * @var InvoiceId|ExpenseId
+     * @JMS\Type("int")
+     * @JMS\SerializedName("object_id")
+     * @JMS\Groups({"request", "response"})
+     */
+    private $objectId;
+
+    /**
+     * @var float
+     * @JMS\Type("double")
+     * @JMS\SerializedName("value")
+     * @JMS\Groups({"request", "response"})
+     */
+    private $value;
+
+    /**
+     * @var float
+     * @JMS\Type("double")
+     * @JMS\SerializedName("social")
+     * @JMS\Groups({"response"})
+     */
+    private $social;
+
+    /**
+     * @var float
+     * @JMS\Type("double")
+     * @JMS\SerializedName("health")
+     * @JMS\Groups({"response"})
+     */
+    private $health;
+
+    /**
+     * @var float
+     * @JMS\Type("double")
+     * @JMS\SerializedName("labor_fund")
+     * @JMS\Groups({"response"})
+     */
+    private $laborFund;
+
+    /**
+     * @var float
+     * @JMS\Type("double")
+     * @JMS\SerializedName("transitional_retiring_fund")
+     * @JMS\Groups({"response"})
+     */
+    private $transitionalRetiringFund;
+
+    /**
+     * @var \DateTime
+     * @JMS\Type("DateTime<'Y-m-d'>")
+     * @JMS\SerializedName("date")
+     * @JMS\XmlElement(cdata=false)
+     * @JMS\Groups({"request", "response"})
+     */
+    private $date;
+
+    /**
+     * @var int
+     * @JMS\Type("int")
+     * @JMS\SerializedName("initial")
+     * @JMS\Groups({"response"})
+     */
+    private $initial;
+
+    /**
+     * @var string
+     * @JMS\Type("string")
+     * @JMS\SerializedName("type")
+     * @JMS\XmlElement(cdata=false)
+     * @JMS\Groups({"response"})
+     */
+    private $type;
+
+    /**
+     * @var string
+     * @JMS\Type("string")
+     * @JMS\SerializedName("payment_method")
+     * @JMS\XmlElement(cdata=false)
+     * @JMS\Groups({"request", "response"})
+     */
+    private $paymentMethod;
+
+    /**
+     * @var string
+     * @JMS\Type("string")
+     * @JMS\SerializedName("payment_type")
+     * @JMS\XmlElement(cdata=false)
+     * @JMS\Groups({"request", "response"})
+     */
+    private $paymentType;
+
+    /**
+     * @var string
+     * @JMS\Type("string")
+     * @JMS\SerializedName("account")
+     * @JMS\Groups({"request", "response"})
+     */
+    private $account;
+
+    /**
+     * @var float
+     * @JMS\Type("double")
+     * @JMS\SerializedName("value_pln")
+     * @JMS\Groups({"request", "response"})
+     */
+    private $valuePln;
+
+    /**
+     * @var float
+     * @JMS\Type("double")
+     * @JMS\SerializedName("currency_exchange")
```

Comment:
Czy pola `currency_* ` są dostępne w module `payments`? Nie widzę ich w dokumentacji. Jeśli są, to powinny znaleźć się w obiekcie `PaymentAmount`.

Reply:
Tych pól nie ma w dokumentacji, ale są w odpowiedzi z API (de facto "tylko do odczytu") i zawierają informacje o kursie wymiany użytym do księgowania przychodów/wydatków w walucie obcej.